### PR TITLE
Feat add and edit notes

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# Feat: Add Note and Update Note

## Added : 
- Notes-context
- Notes Reducer

## Refactor : 
- Note Card component
- Input Feild component

### CODE REVIEW

File: src/Components/InputField/NotesField.jsx [hunk 1]
```diff
@@ -1,33 +1,72 @@
...
+    setNotes((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+  const [isExpanded, setIsExpanded] = useState(false);
+  const expandHandler = () => {
+    setIsExpanded(true);
+  };
+  //onClick={(e) => updatehandler(e, notes)}
```
Comment (Balakumar3018) [inline src/Components/InputField/NotesField.jsx:19]: Remove commented codes

File: src/Pages/HomePage/Homepage.jsx [hunk 2]
```diff
@@ -1,12 +1,89 @@
...
+          },
+        }
+      );
+      noteDispatch({
+        type: "UPDATE_NOTE",
+        payload: resp.data.notes,
+      });
+      setNotes(initialNotes);
+    } catch (error) {
+      console.log(error.message);
```
Comment (Balakumar3018) [inline src/Pages/HomePage/Homepage.jsx:70]: instead of consoling this you can show the error message into view

File: src/Pages/Login/Login.css [hunk 3]
```diff
@@ -20,24 +26,33 @@
...
+  color: var(--font-color);
 }
 .btn-solid {
-  background-color: var(--dark-primary-color);
   width: 100%;
   text-transform: capitalize;
+  background-color: var(--add-btn-color) !important;
+}
+.login-btn {
+  /* background-color: var(--black) !important; */
```
Comment (Balakumar3018) [inline src/Pages/Login/Login.css:54]: remove commented code

**Review by Balakumar3018 — APPROVED** [review]
Good to merge.

File: src/Pages/HomePage/Homepage.jsx (hunk 2 above)
Comment (dusariudaykumar) [reply src/Pages/HomePage/Homepage.jsx:70]: sure, I will add this

**Review by bishnoimukesh — COMMENTED** [review]
Everything looks good.
Suggested to avoid removing and rewriting the same chunk of code and remove the console statements while pushing the code.

**Comment by athar05** [comment]
The UI looks amazing. The transition is also really smooth. Going forward, would love to see a light version of the app as well.

**Pull request merged**